### PR TITLE
Module7 ci

### PR DESCRIPTION
**Pull request by LouisLon**

### CODE REVIEW

File: todo_app/tests_e2e/app_e2e_test.py [hunk 1]
```diff
@@ -26,9 +26,18 @@ def test_app():
     thread.join(1)
     #session.delete_trello_board(board_id)
 
-@pytest.fixture(scope="module")
+# @pytest.fixture(scope="module")
```
Comment (JackMeadDev) [inline todo_app/tests_e2e/app_e2e_test.py]: If we're switching to chromedriver, let's delete this

File: Dockerfile [hunk 2]
```diff
@@ -23,4 +23,29 @@ COPY poetry.lock pyproject.toml /app/
...
+# Install Chromium WebDriver
+RUN LATEST=`curl -sSL https://chromedriver.storage.googleapis.com/LATEST_RELEASE` \
+&& echo "Installing chromium webdriver version ${LATEST}" \
+&& wget https://chromedriver.storage.googleapis.com/${LATEST}/chromedriver_linux64.zip \
+&& unzip ./chromedriver_linux64.zip
+# Install poetry
+#WORKDIR /app
+ENV FLASK_RUN_HOST 0.0.0.0
+ENV FLASK_APP todo_app/tests/app_test.py
+ENV FLASK_ENV development
```
Comment (JackMeadDev) [inline Dockerfile]: We shouldn't need these environment variables, since our tests define any relevant config

File: Dockerfile [hunk 3]
```diff
@@ -23,4 +23,29 @@ COPY poetry.lock pyproject.toml /app/
...
+RUN pip install poetry && poetry config virtualenvs.create true
+COPY poetry.lock pyproject.toml /app/
+RUN poetry install -n
+EXPOSE 5000
```
Comment (JackMeadDev) [inline Dockerfile]: Similarly, this test isn't expecting to expose anything outside the container, so we don't need this line

**Review by JackMeadDev — APPROVED** [review]
Nice, I can see your tests running & passing on travis-ci.org - it's worth noting that that is being sunsetted in favour of travis-ci.com, where your tests are also currently running but [failing](https://travis-ci.com/github/LouisLon/DevOps-Course-Starter/builds/219020431). Looks like that's because you haven't set the environment variables there, so before merging, you should add the relevant variables on that site too, and I'd suggest disabling travis-ci.org once that's working so that we have one source of truth for the tests.

Otherwise, I've left a few thoughts below, but happy for you to merge this when you're ready

**Comment by LouisLon** [comment]
Count not run on https://travis-ci.com/ as it does not pick up the LouisLon/DevOps-Course-Starter repository

**Pull request merged**